environment.yml

```diff
@@ -16,4 +16,7 @@ dependencies:
   - pip=24.3.1
   - click=8.1.7
   - quarto=1.5.57
-  - tabulate=0.9.0
\ No newline at end of file
+  - tabulate=0.9.0
+  - deepchecks=0.18.1
```

Comment:
It may work as it is currently, however after scrolling Slack convos some more I see that it should be nested within pip:
<img width="495" alt="image" src="https://github.com/user-attachments/assets/d84236c1-4ce4-4a40-8bf7-c7c21fd4a73f" />